saged/models.py

```diff
@@ -410,8 +501,6 @@ def fit(self, dataset: LabeledDataset) -> "PytorchSupervised":
             self.loss_fn = self.loss_class(weight=None)
 
         if log_progress:
-            # Set up the neptune experiment
-            utils.initialize_neptune(self.config)
```

Comment:
Is this function still being used anywhere?

Reply:
Yeah, I moved it to `all_labels_comparison`, with the idea that each benchmark will be responsible for setting up its own connection with neptune. It also makes more it so that it's unnecessary to pass all the neptune configuration information to every model